includes/payment-methods/class-wc-stripe-upe-payment-method-multibanco.php

```diff
@@ -21,6 +21,7 @@ public function __construct() {
 		$this->title                = __( 'Multibanco', 'woocommerce-gateway-stripe' );
 		$this->is_reusable          = false;
 		$this->supported_currencies = [ 'EUR' ];
+		$this->supported_countries  = [ 'AT', 'BE', 'BG', 'HR', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GI', 'GR', 'HU', 'IE', 'IT', 'LV', 'LI', 'LT', 'LU', 'MT', 'NL', 'NO', 'PL', 'PT', 'RO', 'SK', 'SI', 'ES', 'SE', 'CH', 'GB', 'US' ];
```

Comment:
<img width="795" alt="Screenshot 2024-07-16 at 2 02 16 PM" src="https://github.com/user-attachments/assets/77ae9a87-deb5-4642-917d-3c0850674db3">

Confirmed that this list matches. 👍